Index.html

```diff
@@ -1,231 +1,328 @@
+						since the 1500s, when an unknown printer took a standard dummy text.
+					</p>
+					<div class="programmingLanguages">
+						<ul class="languagesUsed">
+							<li>css</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>html</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>bootstrap</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>Ruby</li>
+						</ul>
+					</div>
+					<button class="projectButton" type="Button">See Project</button>
+				</div>
+				<div id="project3">
+					<a href="#">
+						<img
+							src="./assets/Img-Placeholder.svg"
+							alt="Screenshot of Project"
+						/>
+					</a>
+					<h3 class="multiPost">Multi-Post Stories</h3>
+					<p class="projectBrief">
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required. has been the industry's standard dummy text ever
+						since the 1500s, when an unknown printer took a standard dummy text.
+					</p>
+					<div class="programmingLanguages">
+						<ul class="languagesUsed">
+							<li>css</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>html</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>bootstrap</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>Ruby</li>
+						</ul>
+					</div>
+					<button class="projectButton" type="Button">See Project</button>
+				</div>
+				<div id="project4">
+					<a href="#">
+						<img
+							src="./assets/Img-Placeholder.svg"
+							alt="Screenshot of Project"
+						/>
+					</a>
+					<h3 class="multiPost">Multi-Post Stories</h3>
+					<p class="projectBrief">
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required. has been the industry's standard dummy text ever
+						since the 1500s, when an unknown printer took a standard dummy text.
+					</p>
+					<div class="programmingLanguages">
+						<ul class="languagesUsed">
+							<li>css</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>html</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>bootstrap</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>Ruby</li>
+						</ul>
+					</div>
+					<button class="projectButton" type="Button">See Project</button>
+				</div>
+			</div>
+			<img class="line" src="./assets/vectorLine.svg" alt="horizontal line" />
+		</section>
+		<footer class="contact">
+			<div class="contact-text">
+				<h2>CONTACT US</h2>
+				<p>
+					If you have an appication you are interested in developing, a feature
+					that you need built or a pooject that needs coding, i'd love to help
+					with it
+				</p>
+			</div>
+			<form action="https://formspree.io/f/meqnjrwb" method="post">
```

Comment:
I have noticed that the form children do not have a consistent font with the rest of the portfolio design. It would be nice 🙏🏻 if the font matched throughout the portfolio as seen in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) to give it a professional look.

Reply:
Alright. Well noted. I will fix that.